js/capture.js

```diff
@@ -10,10 +10,22 @@ const capture = {
   addListeners() {
     // listen for each HTTP response
     browser.webRequest.onResponseStarted.addListener(
-      this.sendThirdParty,
+      this.sendThirdParty.bind(this),
```

Comment:
I prefer:
```
(tabId, changeInfo, tab) => {
  this.sendThirdParty(tabId, changeInfo, tab);
}
```
or
```
(...args) => {
  this.sendThirdParty(...args);
}
```

over:
```
this.sendThirdParty.bind(this)
```

As you aren't creating a new function, there aren't really perf benefits I don't think but it just feels more explicit.